networkx/algorithms/tests/test_euler.py

```diff
@@ -140,16 +140,6 @@ def test_has_eulerian_path_directed_graph(self):
         G.add_edges_from([(0, 1), (1, 2), (0, 2)])
         assert not nx.has_eulerian_path(G)
 
-    def test_has_eulerian_path_isolated_node(self):
-        # Test directed graphs without isolated node returns True
-        G = nx.DiGraph()
-        G.add_edges_from([(0, 1), (1, 2), (2, 0)])
-        assert nx.has_eulerian_path(G)
-
-        # Test directed graphs with isolated node returns True
-        G.add_node(3)
-        assert nx.has_eulerian_path(G)
-
```

Comment:
Let's leave the test code in but change the result tested for with the isolated node to `returns False` and then `assert not ...`.   I think the name of the function no longer makes sense either. It is tempting to just include these in the previous test function -- that is, just take out the function signature line. At least the name of that function describes what we are testing. Some people like to have a separate function for each test, so if you can come up with a good name for it, that is a reasonable approach too.  My desire is just to have the tests to ensure that we know what we want to happen.

Thanks for this!  Nice work...